nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -163,7 +163,11 @@
 - module-name: 'arcade' # checksum: 3f9f0b7d
   data-files:
     dirs:
-      - 'resources'
+      - 'resources/system/'
+  anti-bloat:
+    - no-auto-follow:
+        'examples': 'Arcade examples are not be available unless you do "--include-module=acade.examples"'
```

Comment:
Is arcade really doing imports of those, I seem to recall you saying it does not, then you just don't need to handle that. I understand that for experimental it might, there it makes sense to have it then.